bitrise.yml

```diff
@@ -1418,6 +1418,62 @@ workflows:
       bitrise.io:
         stack: osx-xcode-14.1.x-ventura
         machine_type_id: g2.8core
+  perfherder-test:
+    steps:
+    - cache-pull@2.1:
+        is_always_run: true
+    - xcode-build-for-simulator@0.12:
+        inputs:
+        - test_plan: PerformanceTestPlan
+        - xcodebuild_options: CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO
+        - scheme: Fennec_Enterprise_XCUITests
+    - xcode-test@4.5:
+        inputs:
+        - scheme: Fennec_Enterprise_XCUITests
+        - xcodebuild_test_options: "-testPlan PerformanceTestPlan"
+        - simulator_device: "iPhone 14"
+        - test_plan: PerformanceTestPlan
+    
+    - script@1.1:
+        title: Create perfherder data object
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # debug log
+            set -x
+            # get dependency
+            cd ./test-fixtures && git clone https://github.com/isabelrios/xcresult_extract.git
+            
+            echo $BITRISE_XCRESULT_PATH
+            ls
+            ls $BITRISE_XCRESULT_PATH
+
+            cp -r $BITRISE_XCRESULT_PATH /Users/vagrant/Library/Developer/Xcode/DerivedData/**/Logs/Test/
+            echo "DerivedData/Client/Log/Tests"
+            ls /Users/vagrant/Library/Developer/Xcode/DerivedData
```

Comment:
After this line we should add:
`client_path=$(ls /Users/vagrant/Library/Developer/Xcode/DerivedData | grep 'Client-')`